examples/dune

```diff
@@ -0,0 +1,8 @@
+(executables
```

Comment:
Since there's only one of them we can probably use an `executable` stanza here.

Unless I'm missing something of course!